pages/design/rule-template.md

```diff
@@ -6,41 +6,44 @@ The rule template contains a plain language description of the rule, some backgr
 
 Use the [empty atomic rule template](atomic-template-empty.html) to create new rule. When creating a new rule, first read [rule design](rule-design.html).
 
-## Atomic Rule template
+## Atomic Rule Template
```

Comment:
Would it be possible to embed the template file, so it doesn't have to maintained in two different places? 
They have previously been a bit out of sync, which is confusing.

Reply:
Probably, that seems separate from this pull request.